src/components/LanguageDropdown/index.tsx

```diff
@@ -2,18 +2,26 @@ import React, { useEffect, useRef, useState } from 'react'
 import { useRouter } from 'next/router'
 import Image from 'next/image'
 import LanguageItem from '../LanguageItem'
+import { ComponentType } from '../Footer'
 
 export enum LocaleCode {
   English = 'EN',
-  Thai = 'TH'
+  Thai = 'TH',
 }
 
-const NavbarLanguage = () => {
+interface LanguageDropdown {
+  readonly componentType: ComponentType
+}
+
+const LanguageDropdown = ({ componentType }: LanguageDropdown) => {
+  const router = useRouter()
+  const { locales, locale } = router
```

Comment:
Would you mind if I ask, is `locales` a plural form or a different one?

Reply:
Sure, `locales` in this case is the collection of all locales we have, but `locale` (without `s`) is the current one applying to our website. 
Read more [here](https://nextjs.org/docs/api-reference/next/router)

This is the output:

<img width="164" alt="Screen Shot 2564-02-26 at 00 49 25" src="https://user-images.githubusercontent.com/32285706/109195540-3bc99600-77cd-11eb-886d-d91edcf87c20.png">